suite-native/module-authorize-device/src/components/connect/DeviceConnectionScreenWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode, useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import { useFocusEffect } from '@react-navigation/native';
+
+import { selectIsDeviceAuthorized } from '@suite-common/wallet-core';
+import { Screen, useNavigateToInitialScreen } from '@suite-native/navigation';
+import TrezorConnect, { DEVICE } from '@trezor/connect';
+
+import { ConnectDeviceScreenHeader } from './ConnectDeviceScreenHeader';
+
+export const DeviceConnectionScreenWrapper = ({
```

Comment:
This is in fact a `Screen`:
```suggestion
export const DeviceConnectionScreen = ({
```

Reply:
Fixed